.vscode/settings.json

```diff
@@ -9,7 +9,7 @@
         "-s",
         "./tests",
         "-p",
-        "test_*.py"
+        "*test.py"
```

Comment:
I'm concerned that this will affect the testing files used. 
Could this be reverted?